judge/migrations/0099_partially_hide_scoreboard.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.9 on 2019-12-27 20:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0098_registration_data'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='contest',
+            name='partially_hide_scoreboard',
+            field=models.BooleanField(default=False, help_text="Whether the scoreboard should be shown remain hidden until the member's contest time is finished", verbose_name='Partially hide scoreboard'),
```

Comment:
```suggestion
            field=models.BooleanField(default=False, help_text="Whether the scoreboard should be hidden until the member's contest time is finished", verbose_name='Partially hide scoreboard'),
```

Reply:
Actually this entire `help_text` doesn't match the model's.

Reply:
How about `Whether the scoreboard is hidden until the user finishes their contest window`

Reply:
Let's go with:
> Whether the scoreboard is hidden until the user's contest window ends

Make sure to change it in the model and the migration.